tests/graph/test-data/rust/src/seeding/distributions/adaptors/uniform.rs

```diff
@@ -48,7 +48,7 @@ where
     fn create_distribution(&self) -> Result<Self::Distribution, Self::Error> {
         Ok(UniformDistribution {
             index: Uniform::new(0, self.distributions.len())
-                .expect("`Uniform` should return a valid index"),
+                .change_context(UniformDistributionConfigError::Distribution)?,
```

Comment:
error conditions are (https://docs.rs/rand/latest/rand/distr/struct.Uniform.html#method.new_inclusive):
- low > high
- low is non-finite
- high is non-finite
- high - low is non-finite

In this case _none_ of these cases can happen, so I think an `unwrap_or_else(|_err| unreachable!())` would be more fitting.